### PR TITLE
dist: add systemd service unit

### PR DESCRIPTION
**Pull request by lucab**

### CODE REVIEW

File: dist/systemd/system/zincati.service [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Zincati update agent
```
Comment (jlebon) [inline dist/systemd/system/zincati.service]: This is minor, though I think the convention leans a bit more on the description being Title Case?

Comment (jlebon) [inline dist/systemd/system/zincati.service]: Maybe let's also add for now `Documentation=https://github.com/coreos/zincati` ?

File: dist/systemd/system/zincati.service [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Requires=rpm-ostreed.service
+After=rpm-ostreed.service
```
Comment (jlebon) [inline dist/systemd/system/zincati.service]: I don't think we need these. The daemon isn't long-running; it auto-exits when there are no more clients connected.

**Comment by lucab** [comment]
Amended.

**Pull request merged**